tests/drivers/adc/adc_api/boards/cyw920829m2evk_02.conf

```diff
@@ -0,0 +1 @@
+CONFIG_TEST_USERSPACE=n
```

Comment:
Why?

Reply:
This was originally done to avoid a fault that was occurring when running in userspace. This has now been addressed in this PR with modifications to mpu.